app/models/pseud.rb

```diff
@@ -268,6 +287,11 @@ def self.parse_bylines(list, options = {})
         pseuds = pseuds - banned_pseuds
         banned_pseuds = banned_pseuds.map(&:byline)
       end
+      disallowed_pseuds = pseuds.reject { |pseud| Pseud.check_pseud_coauthor?(pseud.id)}
+      if disallowed_pseuds.present? && options[:remove_disallowed]
+        pseuds = pseuds - disallowed_pseuds
```

Comment:
I agree this happens, going to ask what we want to do about it

Reply:
sarken
"because the person entering it has no way of knowing if the person they're trying to add has the setting enabled. If I enter testy and the user testy doesn't allow co-creators, but testy (testy2) does, I super don't want it to add the wrong person"

Reply:
This has been fixed in a later version